src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/RuntimeIdentifierTests.cs

```diff
@@ -15,7 +15,7 @@ public void VerifyOSRid()
         {
             Assert.NotNull(RuntimeInformation.RuntimeIdentifier);
             Assert.Same(RuntimeInformation.RuntimeIdentifier, RuntimeInformation.RuntimeIdentifier);
-            Assert.EndsWith(RuntimeInformation.ProcessArchitecture.ToString(), RuntimeInformation.RuntimeIdentifier, StringComparison.OrdinalIgnoreCase);
+            Assert.EndsWith(RuntimeInformation.ProcessArchitecture.ToString(), PlatformDetection.IsNotBrowser ? RuntimeInformation.RuntimeIdentifier : "browser-wasm", StringComparison.OrdinalIgnoreCase);
```

Comment:
we should have a tracking issue to revert this once `RuntimeInformation.RuntimeIdentifier` returns the right info.
or maybe it makes more sense to use `[ActiveIssue]` for that.

Reply:
I like `[ActiveIssue]` better.

Reply:
Done